src/core/lib/security/security_connector/security_connector.h

```diff
@@ -243,6 +243,7 @@ grpc_auth_context* tsi_ssl_peer_to_auth_context(const tsi_peer* peer);
 tsi_peer tsi_shallow_peer_from_ssl_auth_context(
     const grpc_auth_context* auth_context);
 void tsi_shallow_peer_destruct(tsi_peer* peer);
+int ssl_host_matches_name(const tsi_peer* peer, const char* peer_name);
```

Comment:
If this function is going to be exposed (even if just for testing), it should have a `grpc_` prefix, to avoid polluting the global symbol namespace.

Reply:
what about promoting it to be part of the `tsi`?

Reply:
@jiangtaoli2016's team owns the TSI API, so he can speak to whether he wants this in that namespace.

Note that the existing `tsi_*` functions in this header file are IMHO either incorrectly named or else are in the wrong place.  If they are actually part of TSI, they belong somewhere in `src/core/tsi`; otherwise, they should be named with a `grpc_` prefix instead of a `tsi_` prefix.  But this is a pre-existing problem, not something you should need to fix in this PR.

Reply:
It is pre-existing. 
@dcow how about you change to grpc_ prefix on this particular function, i.e., grpc_ssl_host_matches_name.

I will have a separate PR to clean up the rest.